### ISSUE TITLE
[lldb-vscode] Feature Request: more disassembler functionality

### ISSUE THREAD

**Issue by d-mozulyov** [llvm/llvm-project#62545]
### Problem
When developing complex high-performance code, it can be extremely useful to see what assembler is generated in a real project (for example, in RelWithDebInfo mode). Now in lldb-vscode there is no way to look at the disassembler at an arbitrary point in time, you can't put a breakpoint on any line of the disassembler, you can't look at the assembler in Intel syntax.

![disasm_small](https://user-images.githubusercontent.com/10961593/236158619-d5d739d3-802f-4dc9-a923-26f8b4efa16f.png)

### Conclusions
I suggest looking at the experience of other IDEs, such as C++ Builder (maybe QtCreator, etc. has similar functionality), which, by the way, is based on Clang.
1. Provide the ability to call the disassembler window at an arbitrary moment (I think this is possible in CodeLLDB)
2. Make Intel syntax default or provide a choice
3. Provide the ability to put breakpoints on any assembler instruction

![disassembly](https://user-images.githubusercontent.com/10961593/236158635-a1e6acbe-56e7-44e8-a53d-447aeb3906a1.png)

**Comment by asl** [comment]
Probably it should be reported at https://github.com/lanza/lldb-vscode not here

**Comment by d-mozulyov** [comment]
@asl 

I do not agree with your opinion. The repository that you have indicated has not been developed for several years and is not directly related to LLVM.

**Comment by asl** [comment]
@jryans @d-mozulyov Yeah, sorry. For some reason I somehow missed it at first.
